app/player/player.controller.js

```diff
@@ -18,36 +21,64 @@ angular.module('app.player')
 
 	// add player with a name, and a initial score
 	vm.addPlayer = function() {
-		vm.players.push({ 
+		
+		vm.storagePlayers.push({ 
 			'name':vm.name,
-			'totScore':501
+			'totScore':201,
+			'selected':false
 		});
-  	vm.name = ''; //clear input
-	  //save the last player
-		var currentPlayerSaved = {};
-	  currentPlayerSaved = vm.players[vm.players.length - 1]; 
-	  vm.storagePlayers.push(currentPlayerSaved); //add player in the VIP array
-	  $localStorage.players = vm.players;
-	  $localStorage.storagePlayers = vm.storagePlayers;
+		vm.name = ''; //clear input
+		//save the last player
+		//var currentPlayerSaved = {};
+		//currentPlayerSaved = vm.players[vm.players.length - 1]; 
+		//vm.storagePlayers.push(currentPlayerSaved); //add player in the VIP array
+		//$localStorage.players = vm.players;
+		$localStorage.storagePlayers = vm.storagePlayers;
+		addButtonClicked = false;
 	};
 
 	//remove all players in the list
-	vm.removeAllPlayers = function() {
+	/*vm.removeAllPlayers = function() {
 		vm.players = [];
 		$localStorage.players = vm.players;
-	};
+	};*/
 
 	// remove only the VIP player selected
-	vm.removePlayerStorage = function(friend) {
+	/*vm.removePlayerStorage = function(friend) {
 		var index = vm.storagePlayers.indexOf(friend);
 		vm.storagePlayers.splice(index, 1);
 		$localStorage.storagePlayers = vm.storagePlayers;
-	};
+	};*/
 
 	//take the selected VIP player from table and put it in the list of player
-	vm.addPlayerInList = function(friend) {
+	/*vm.addPlayerInList = function(friend) {
 		vm.buttonAddPlayerTable = true;
 		vm.players.push(friend);
 		$localStorage.players = vm.players;
+	};*/
+
+	vm.addPlayerClicked = function() {
+		addButtonClicked = true;
+	};
+
+	vm.isAddPlayerClicked = function() {
+		return addButtonClicked;  
 	};
+
+	vm.slectDeselectPlayer = function(friend) {
```

Comment:
typo...
btw... `selectDeselectPlayer` seems to be not the most appropriate name for it. Why not using something like `togglePlayer`?